CHANGELOG.md

```diff
@@ -6,12 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
-- Move Settings into channel on the homescreen (#689)
 
 ### Changed
 
 ### Fixed
 
+
+## [3.8] 2019-04-30
+### Added
+- Move Settings into channel on the homescreen (#689)
+
+### Fixed
+- Fix Pocket timeout exceptions (#2095)
```

Comment:
I'm surprised you're adding this and they weren't already added 🤔 : is this a mistake? I can't verify the correctness of adding these.

Reply:
I was looking through the commits and didn't see #2095 listed, and it's a fix so I added it in.